enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -768,7 +770,7 @@ def can_redeem(self, request, enterprise_customer_uuid):
                     enterprise_customer_uuid, lms_user_id, content_key
                 )
 
-            if not redemptions and not redeemable_policies:
+            if not successful_redemptions and not redeemable_policies:
```

Comment:
In local testing, I noticed I wasn't getting any `reasons` in my test responses even when I can an expected `can_redeem=False` result. I tend to use reversals a lot to set up test state, so I have many unsuccessful redemptions.

Reply:
This seems reasonable!